packages/ui-components/src/Icon/Icon.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FunctionComponent } from "react";
+import classNames from "classnames/bind";
+import get from "lodash/get";
+
+import * as Icons from "@cockroachlabs/icons";
+import styles from "./Icon.module.scss";
+
+import objectToClassnames from "../utils/objectToClassnames";
+
+type IconSize = "large" | "medium" | "default" | "small" | "tiny";
```

Comment:
I think we usually use t-shirt sizes. It's probably worth checking in with design (if you haven't already) to make sure they use the same names. I don't see different sizes laid out in the DS mocks.

Reply:
I took the props for this component from the existing Icon component in managed service. I will confirm with designer what sizes of icon are preferable.

As far as t-shirt sizes go, do you mean we usually use abbreviations? (like `l`, `m`, `s`, `xs`, etc) I find those to be confusing and wanted to avoid bike-shedding whether to use one letter or two letters or whatever. I find words to be more readable, predictable, and less confusing.

Reply:
@Annebirzin @rsadres Regarding Icon sizes ☝️ Does design have a comprehensive list of sizes for icons? I'm not sure if I saw an audit done for icons.